linetools/spectra/io.py

```diff
@@ -377,9 +401,34 @@ def chk_for_gz(filenm):
         return filenm, chk
 
     # Add .gz
-    if os.path.lexists(filenm+'.gz'): 
+    if os.path.lexists(filenm+'.gz'):
         chk=True
         return filenm+'.gz', chk
     else:
         chk=False
         return None, chk
+
+def is_UVES_popler(hd):
+    """ Check if UVES_popler output.
+    """
+    if 'history' not in hd:
+        return False
+    for row in hd['history']:
+        if 'UVES POst Pipeline Echelle Reduction' in row:
+            return True
+    return False
+
+def parse_UVES_popler(hdulist):
+    """ Read a spectrum from a UVES_popler-style fits file.
+    """
+    from linetools.spectra.xspectrum1d import XSpectrum1D
+
+    hd = hdulist[0].header
+    uwave = setwave(hd) * u.Angstrom
+    co = hdulist[0].data[3]
+    fx = hdulist[0].data[0] * co  #  Flux
```

Comment:
This makes fx equal to the un-normalised flux. I prefer this to be the default - i.e. flux should be un-normalised, and the user can normalise it if desired. But if you strongly prefer the opposite I'm happy to do that too.